scripts/utils/starknet.py

```diff
@@ -246,15 +266,78 @@ def compile_contract(contract):
     if output.returncode != 0:
         raise RuntimeError(output.stderr)
 
+    def _convert_offset_to_hex(obj):
+        if isinstance(obj, list):
+            for i in range(len(obj)):
+                obj[i] = _convert_offset_to_hex(obj[i])
+        elif isinstance(obj, dict):
+            for key in obj:
+                if obj.get(key) is not None:
+                    obj[key] = _convert_offset_to_hex(obj[key])
+        elif isinstance(obj, int) and obj >= 0:
+            obj = hex(obj)
+        return obj
+
+    compiled = json.loads((BUILD_DIR / f"{contract['contract_name']}.json").read_text())
+    compiled = {
+        **compiled,
+        "entry_points_by_type": _convert_offset_to_hex(
+            compiled["entry_points_by_type"]
+        ),
+    }
+    json.dump(
+        compiled, open(BUILD_DIR / f"{contract['contract_name']}.json", "w"), indent=2
+    )
+
+
+def compress_program(program):
```

Comment:
Are we using `compress_program` and `decompress_program` anywhere? If not can be removed, as well as all imports linked to them (gzip, base64,...).